### PR TITLE
Add option to not cleanup urls when training duplicate model

### LINKED ISSUE #709: Add an option not to cleanup URLs for the Duplicate model
In the duplicate case, having two equal URL can be a strong indication that two bugs are the same.

### PR DESCRIPTION
**Pull request by cklyyung**
Closes #709.

### CODE REVIEW

**Comment by cklyyung** [comment]
If I'm reading this correctly, slightly more duplicates are found when the urls are not cleaned up.

With url cleanup
cleanup_urls

Without url cleanup
dont_cleanup_urls

This is for a training set size of 14000.

**Comment by marco-c** [comment]
@cklyyung the option should actually be added to the similarity script :)
We can add it to the Duplicate model too in the future, but we need to fix #724 first for that.

**Comment by marco-c** [comment]
Well, let's do it for the Duplicate model too anyway, and we'll block adding new additional options on fixing #724.

File: run.py [hunk 1]
```diff
@@ -29,6 +29,13 @@ def parse_args(args):
         type=int,
         help="The size of the training set for the duplicate model",
     )
+    parser.add_argument(
+        "--dont-cleanup-urls",
```
Comment (marco-c) [inline run.py]: Nit: `--disable-url-cleanup`

**Comment by cklyyung** [comment]
> @cklyyung the option should actually be added to the similarity script :)
> We can add it to the Duplicate model too in the future, but we need to fix #724 first for that.

Ah shoot, sorry my bad.

**Pull request merged**